storage/Storage.ts

```diff
@@ -56,6 +56,7 @@ export default class Storage {
           if (version === 0) {
             upgrade.createStore("lastModified");
             upgrade.createStore("property");
+            upgrade.createStore("household");
```

Comment:
I think we need to start bumping the version now, since we're deploying somewhere non-developers are interacting with it.